CTD.py

```diff
@@ -0,0 +1,220 @@
+from collections import Counter
+import argparse
+from scipy.stats import norm
+from sklearn import covariance
+import json
+from multiprocessing import Pool
+from functools import partial
+import pandas as pd
+from Python import data, graph, mle
+import logging
+import numpy as np
+import os
```

Comment:
Add imports from the standard Python library first, then empty line then other, from pip-installed libraries